src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -38,31 +38,27 @@
   @Parameter(property = "cloudSdkPath", required = false)
   protected File cloudSdkPath;
 
-  protected CloudSdk.Builder cloudSdkBuilder;
-
-  private final ProcessOutputLineListener gcloudOutputListener = new ProcessOutputLineListener() {
+  protected final ProcessOutputLineListener gcloudOutputListener = new ProcessOutputLineListener() {
```

Comment:
I don't really have a preference, but willing to change, if there is some best-practice that it's violating.